src/main/java/com/github/jparkie/pdd/BitArray.java

```diff
@@ -38,22 +38,35 @@ private static int numWords(long numBits) {
         return (int) numWords;
     }
 
+    private final class indexAndMask {
```

Comment:
Oh, yeah... I guess I keep crossing the streams... eliminated the class entirely.